pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -37,11 +37,13 @@ const (
 )
 
 type StorageInitializerConfig struct {
-	Image         string `json:"image"`
-	CpuRequest    string `json:"cpuRequest"`
-	CpuLimit      string `json:"cpuLimit"`
-	MemoryRequest string `json:"memoryRequest"`
-	MemoryLimit   string `json:"memoryLimit"`
+	Image                string `json:"image"`
+	CpuRequest           string `json:"cpuRequest"`
+	CpuLimit             string `json:"cpuLimit"`
+	MemoryRequest        string `json:"memoryRequest"`
+	MemoryLimit          string `json:"memoryLimit"`
+	StorageSpecSecret    string `json:"storageSpecSecret"`
+	DefaultCredentialKey string `json:"defaultCredentialKey"`
```

Comment:
I think we should rename this to something like `DefaultStorageSecretKey` or `DefaultStorageConfigKey`

Reply:
renamed to `DefaultStorageSecretKey`